packages/modules-sdk/src/definitions.ts

```diff
@@ -11,6 +11,7 @@ export enum Modules {
   CACHE = "cacheService",
   PRODUCT = "productService",
   PRICING = "pricingService",
+  PROMOTION = "promotionService",
```

Comment:
todo: I think we should start to name the modules with the `Service` suffix.
```suggestion
  PROMOTION = "promotion",
```

Reply:
can you elaborate on this? Its going against the other modules' convention

Reply:
Instead of having the `Service` suffix on module names, we should follow a convention similar to that of the package name. 

You would use this name to install the module in `medusa-config.js` and is not really related to anything about services right? Please correct me if I am wrong.

So it would be:
```
{
  promotion: {
    resolve: "...",
    options: { ... }
  }
}
```

instead of:
```
{
  promotionService: {
    resolve: "...",
    options: { ... }
  }
}
```
Which reads a bit nicer.

But yes, this would break the convention meaning we would have to update all the other module names for Medusa 2.0

Reply:
That said, fine to keep as is and revisit later.

Reply:
No, you're right. I prefer this much more to the one with service.